internal/dispatch/graph/reachableresources_test.go

```diff
@@ -603,3 +609,109 @@ func TestCaveatedReachableResources(t *testing.T) {
 		})
 	}
 }
+
+func TestReachableResourcesWithConsistencyLimitOf1(t *testing.T) {
```

Comment:
consistency -> concurrency?

```suggestion
func TestReachableResourcesWithConcurencyLimitOf1(t *testing.T) {
```